tests/KlaviyoAPITest.php

```diff
@@ -370,7 +364,7 @@ public function returnFormatCurlHeaders( $headers )
         };
 
         $headers = array(Klaviyo::USER_AGENT => 'Klaviyo-PHP/' . Klaviyo::PACKAGE_VERSION);
-        $expected = array('User-Agent: Klaviyo-PHP/2.1.0');
+        $expected = array('User-Agent: Klaviyo-PHP/2.1.2');
```

Comment:
can we update the expected to also use the package version constant? otherwise we're never going to remember to update this test each time and it's a waste of time

Reply:
you got it, updated